src/Sound.cpp

```diff
@@ -0,0 +1,37 @@
+#include "Sound.hpp"
+
+Sound::Sound() { /* empty */ }
+
+Sound::Sound(const Sound& sound){
+  mBuffer = sound.mBuffer;
+  mSound = sound.mSound;
+  set_buffer();
+}
+
+Sound::Sound(std::string file_name) {
+  if (load_from_file(file_name)) {
```

Comment:
We are silently ignore an error here. I believe we should throw an exception in case we cannot load the sound file correctly.